autrainer/datasets/aibo.py

```diff
@@ -110,16 +111,50 @@ def __init__(
                 self.df_train, self.train_transform
             )
 
-    def load_dataframes(
-        self,
-    ) -> Tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame]:
-        return (
-            pd.read_csv(
-                os.path.join(self.path, f"train_{self.aibo_task}.csv")
-            ),
-            pd.read_csv(os.path.join(self.path, f"dev_{self.aibo_task}.csv")),
-            pd.read_csv(os.path.join(self.path, f"test_{self.aibo_task}.csv")),
+    @property
+    def audio_subdir(self):
+        """Subfolder containing audio data."""
+        return "wav"
+
+    @cached_property
+    def _load_df(self, path):
```

Comment:
Oh and also here and below type annotations are missing and the `_load_df` call is missing `path`.